pym/gentoolkit/equery/meta.py

```diff
@@ -189,10 +189,43 @@ def _format_upstream_docs(docs):
 
     def _format_upstream_ids(ids):
         result = []
+        sites = {
```

Comment:
```suggestion
        remote_ids = {
```

Reply:
or maybe: `remote_id_map`? not sure

Reply:
remote_ids was available so went with that although there's one more var remoteids.. should I change it to remote_id_map ?